Delete_and_Earn(740).py

```diff
@@ -0,0 +1,27 @@
+# Time Complexity : O(n)
+# Space Complexity : O(1)
```

Comment:
Recheck space complexity

Reply:
Thank You for pointing it out.
It would be O(n) since an extra array is being used after preprocessing.
I have made the correction and pushed the code now.